src/pool/disjoint/pool_disjoint_config_parser.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2023 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef USM_POOL_CONFIG
+#define USM_POOL_CONFIG
+
+#include "pool_disjoint.hpp"
+
+#include <string>
+#include <unordered_map>
+
```

Comment:
Hm, actually, I think we should not move the pool_disjoint_config_parser.* files.

The parseDisjointPoolConfig is called by UR, and if this is implemented here, we would have to expose this function in the API, which I think we don't want. 

@pbalcer @bratpiorka what do you think? Should we keep those files in the UR?

Reply:
Removed